web-api/src/net.rs

```diff
@@ -0,0 +1,183 @@
+// Copyright 2022 Xayn AG
```

Comment:
This file is basically [web-api/src/server/config.rs](https://github.com/xaynetwork/xayn_discovery_engine/pull/797/files#diff-47f02916c7b3ffff14ac94561d1f7071709ddcc9ca1c8a7be2e137a3e7f9b61b) with an additional `start_actix_server` function, separating most "net" related server setup (bind and tcp option setup, start/stopping listening, transport compression, etc.)